chromium_src/chrome/browser/android/tab_features.h

```diff
@@ -12,6 +12,17 @@
 
 #undef TabFeatures
 
-#include "brave/browser/android/tab_features.h"
+namespace tabs {
+class BraveTabFeatures;
+class TabFeatures {
```

Comment:
this is admittedly not ideal, but not sure how else to handle it unless we add a static create method to upstream to match desktop